Source/WebKit/UIProcess/Cocoa/WebPageProxyCocoa.mm

```diff
@@ -978,6 +978,9 @@ static bool exceedsRenderTreeSizeSizeThreshold(uint64_t thresholdSize, uint64_t
     if (preferences().useGPUProcessForDOMRenderingEnabled())
         return true;
 
+    if (m_configuration->lockdownModeEnabled())
```

Comment:
This will affect down levels, and we don't want it to.

Reply:
> This will affect down levels, and we don't want it to.

Ah, good catch! Fixed in latest patch.

Thanks for reviewing!